README.md

```diff
@@ -68,23 +70,32 @@ or
 yarn add @guardian/cdk
 ```
 
-Patterns can be imported from the top level of the library (e.g. `import { InstanceRole } from "@guardian/cdk";`) while constructs must be imported from their construct directory (e.g. `import { GuAutoScalingGroup } from "@guardian/cdk/lib/constructs/autoscaling";`)
+Patterns can be imported from the top level of the library:
 
-There are more details on using the CDK library in [docs](docs)
+```typescript
+import { GuScheduledLambda } from "@guardian/cdk";
+```
 
-## Releasing
+While constructs must be imported from their construct directory:
```

Comment:
Perhaps we should repeat the point about using patterns over constructs here (in case people skip straight to the `Usage` section?

```suggestion
We encourage you to use patterns rather than constructs whenever possible. If you need to use a construct directly, they must be imported from their construct directory:
```